docs/_attic/glossary_entries.py

```diff
@@ -241,6 +254,13 @@
 	institute="Dockstore", 
 	pronunciation='')
 
+EnvironmentVariable = GlossEntry("environment variable", 
+	acronym_full="", 
+	definition="A variable that affects how processes run on a computer. For example, [cwltool] references the environment variable $TMPDIR when deciding where to place files. You can set environment variables in Bash using the `export` command. Environment variables are sometimes called env var or env for short.", 
```

Comment:
"...are sometimes called env var..." -- I've never heard that, but it might just be me.

Reply:
(also no) 
https://books.google.com/ngrams/graph?content=%22env+var%22%2C%22environment+variables%22&year_start=1800&year_end=2019&corpus=26&smoothing=3

Reply:
I've seen "env variables" and "environment vars" and "envs" -- wanted to get all of those across. But yeah, now that I think about it I've never seen "env vars." Is the shorthand self-evident enough for me to get rid of that bit or should I list the variations that do get use?

Reply:
I think environment variables is probably enough. If they've heard of one of the more unusual variations, hopefully they already learned what it is.